android-kotlin/app/src/main/res/values/strings.xml

```diff
@@ -8,4 +8,5 @@
     <string name="already_have_account">Already have an account? Log in</string>
     <string name="does_not_have_account">Don\'t have an account? Create account</string>
     <string name="log_out">Log Out</string>
+    <string name="more">\u22EE</string>
```

Comment:
If we plan on popping up a menu here, this is fine, but if we don't have any other interaction plans for this button we should just change the icon to a trash can or an X, something that makes it obviously a delete interaction. particularly because deleting the item is NOT what most users would expect from a triple-dot menu.

Reply:
true, I was planning to change it to an X if I couldn't get the pop up menu to work